promgen/management/commands/targets.py

```diff
@@ -12,12 +12,12 @@
 class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument('--reload', action='store_true',
```

Comment:
I tried using ```black``` instead of ```autopep8```. https://github.com/line/promgen/pull/212/commits/5e258cf2f81dfd0b0ecc623a30b3fe84b98c353f
If this is ok, try applying ```black``` to other files as well?